src/random_number.py

```diff
@@ -0,0 +1,4 @@
+from random import randint
+
+def random_number(min: int, max: int) -> int:
```

Comment:
Il nome delle variabili in questa maniera e' estremamente esplicito, pero' c'e' il problema che vanno a sovrascrivere (e quindi nascondere) le funzioni built-in di python, `max` e `min`, che quindi non potresti andare ad utilizzare all'interno della funzione.
Potresti quindi optare per nomi alternativi come `min_value` o `minimum` o `puffo`. Magari non l'ultimo. A meno che tu non sia Barbanera.